.github/workflows/e2e_cron.yml

```diff
@@ -0,0 +1,84 @@
+name: Monitor Cron e2e Tests
+on:
+  schedule:
+    - cron: '0 8 * * *'
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to run the e2e against'
+        required: false
+        default: 'stage'
+        type: choice
+        options:
+          - stage
+          - prod
+          - dev
+      type:
+        description: 'What Type of Test do you want to run?'
+        required: false
+        default: 'all'
+        type: choice
+        options:
+            - all
+            - smoke
+            - ui
```

Comment:
Question: in what situations would we want to skip smoke tests?

Reply:
won't ever skip, just want to give stakeholders the ability to choose whether to run all, just smoke, or ui regression for their particular instance

Reply:
Running UI regression tests skip smoke tests, right? I'm just wondering why anyone would ever want to run _only_ those.

Reply:
my original thought on this is flexibility, for all stakeholders however, i guess all should work